## DP: Description: Add a no-op set_irq_affinity_info() for non-SMP
## DP: Patch author: dann frazier <dannf@hp.com>
## DP: Upstream status: Submitted
#
--- linux-2.6-2.6.13/arch/ia64/kernel/irq.c~	2005-10-20 17:28:56.000000000 -0600
+++ linux-2.6-2.6.13/arch/ia64/kernel/irq.c	2005-10-20 17:29:41.000000000 -0600
@@ -105,6 +105,11 @@
 		irq_redir[irq] = (char) (redir & 0xff);
 	}
 }
+
+#else /* !CONFIG_SMP */
+void set_irq_affinity_info (unsigned int irq, int hwid, int redir)
+{
+}
 #endif /* CONFIG_SMP */
 
 #ifdef CONFIG_HOTPLUG_CPU
